src/core/Microsoft.PowerToys.Settings.UI.UnitTests/ViewModelTests/General.cs

```diff
@@ -0,0 +1,228 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO;
+using System.Text.Json;
+using Microsoft.PowerToys.Settings.UI.Lib;
+using Microsoft.PowerToys.Settings.UI.Lib.ViewModels;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace ViewModelTests
+{
+    [TestClass]
+    public class General
+    {
+        [TestInitialize]
+        public void Setup()
+        {
+            // initialize creation of test settings file.
+            GeneralSettings generalSettings = new GeneralSettings();
+            SettingsUtils.SaveSettings(generalSettings.ToJsonString());
+        }
+
+        [TestCleanup]
+        public void CleanUp()
+        {
+            // delete folder created.
+            string generalSettings_file_name = string.Empty;
```

Comment:
I believe this is what is causing the error i mentioned.

Doesn't this seem quite dangerous? It looks like your trying to delete the users settings folder.  Basically meaning running unit tests will nuke anyones settings.   

Also the two if statements appear to do the same thing.

Reply:
The UWP project created a sub-virtual path for the test project files and I assumed it was gonna work the same way on the MSTest project. Ill make the updates.

Reply:
https://github.com/microsoft/PowerToys/pull/5753/commits/37f40bb7a387deb8edd9a9a258ff34ac73d65a4c

Reply:
https://github.com/microsoft/PowerToys/pull/5753/commits/37f40bb7a387deb8edd9a9a258ff34ac73d65a4c